.github/workflows/metadata-check.yml

```
@@ -0,0 +1,46 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: Metadata Checker 
+on:
+  push:
+    branches: 
+      - dev
+      - master
```

So that this gets checked on each PR
```suggestion
  pull_request:
    branches: 
      - dev
```